cypress/integration/comparator.spec.ts

```diff
@@ -1,2 +1,49 @@
 // TODO: implement e2e for comparator here
+
+// Write new test that fills out the form so that it ends up with the standard comparator query, to confirm our form is functional.
+
+// Ask Belco how to get cy.findByText(/Changes from v6.16.0 to v8.1.0/i).should('exist') to wait for the result.
+// We should mock/fixture the result somehow, as we're expecting the same results each time... and it's really easy to hit the API limits unauthenticated.
+// For now these tests do work properly, but they're not using any Testing Library queries.
+
+it('should show expected results when using standard query string', () => {
+  cy.visit(
```

Comment:
I think we should have a fixture/mock of the real response, that we then use to test with.
I'm hitting the API limits within 5 minutes of starting my work on the tests, because cypress auto-runs whenever you change a test... 😄

Reply:
Yes, we definitely need to stub all requests to GitHub in our Cypress tests. Otherwise, we can hit the API limit in the CI and get failing tests.

This is the official way Cypress provides to do so: https://docs.cypress.io/guides/guides/network-requests#Stubbing

It's ok for now to stub those requests per test basics with hardcoded values, and then we can extract some generic fixtures.

Reply:
I can help you with this one if you don't mind me pushing code to your branch.

Just let me know which repository and versions you'd like to use for the test and I can do that in this PR.

Reply:
> I can help you with this one if you don't mind me pushing code to your branch.

Yes, please help me! I've never done this before. I've read the docs, but it's still not clear how I should start.

Feel free to push code to the PR branch, just don't force-push anything. 😉 
I'm not available for the rest of the day, so feel free to work on this, you won't get any new stuff from me today. 😄 

Might be good if you could write down the steps you're doing, so that I can learn how this works, I don't know if seeing the code will be good enough for me to understand the steps. 😉

Reply:
I'll do it then. Not sure if I'll be able to do so today, but during the week for sure.

Reply:
> Just let me know which repository and versions you'd like to use for the test and I can do that in this PR.

I think doing the standard query string would be good, as that has most of the styling we care about in the response. 😉 

```
?repo=testing-library%2Fdom-testing-library&from=v6.16.0&to=v8.1.0
```

Reply:
I had an idea so we have a workaround until I can set the stubs properly (I want to use msw.js and I need to check a couple of things).

I've pushed some changes to this branch so you can use a personal GitHub access token for running the tests in Cypress! I added some instructions to the `CONTRIBUTING.md` file. The CI is already set up with a personal token I generated so it's already running with it, getting an extended rate limit.

Check it out and let me know what you think!

Reply:
At least now we can use a token to increase the amount of API calls. 😄